docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -643,6 +643,34 @@ but you can still access the original via ``hostvars``::
 This works for all overriden connection variables, like ``ansible_user``, ``ansible_port``, etc.
 
 
+.. _scp_protocol_error_filename:
+
+How do I fix 'protocol error: filename does not match request' when fetching a file?
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+When using ``scp`` as the file transfer mechanism for SSH, it can fail with the error::
```

Comment:
I'd start with the reason for the error (esp. since it's an external bug). Something like:
"Newer releases of OpenSSH have a bug in the `scp` client that can trigger this error on the Ansible controller:"